HACKING.md

```diff
@@ -0,0 +1,88 @@
+# Hacking
+
+## `make-prs.py`
+
+The `make-prs.py` script (only exists in the main branch) does the following:
+
+- Get the diff from a PR;
+- find out which chapter that PR is about;
+- create branches (in your forked repo), merge the diff into all following chapters, commit, and push;
+- create PRs to the upstream.
+
+### Configuration
+
+- GITHUB_TOKEN env var is set.
+
+### Dependencies
+
+- `git` CLI
+- `tox`
+- `PyGithub`, `requests` (both are already automatically managed by `tox`)
+
+### Usage
+
+```console
+usage: make-prs.py [-h] [-u UPSTREAM_OWNER] -f FORK_OWNER [-ur UPSTREAM_REPO_NAME]
+                   [-fr FORK_REPO_NAME] -p PULL_REQUEST_NUMBER [-i]
+
+options:
+  -h, --help            show this help message and exit
+  -u UPSTREAM_OWNER, --upstream-owner UPSTREAM_OWNER
+                        The upstream owner of the repository, defaults to `canonical`.
+  -f FORK_OWNER, --fork-owner FORK_OWNER
+                        The forked repo's owner, mandatory. Normally, it should be your own
+                        GitHub user name.
+  -ur UPSTREAM_REPO_NAME, --upstream-repo-name UPSTREAM_REPO_NAME
+                        The repository name in the upstream, defaults to `juju-sdk-tutorial-k8s`.
+  -fr FORK_REPO_NAME, --fork-repo-name FORK_REPO_NAME
+                        The repository name of the forked repo, defaults to `juju-sdk-
+                        tutorial-k8s`.
+  -p PULL_REQUEST_NUMBER, --pull-request-number PULL_REQUEST_NUMBER
+                        The PR number from which you want to merge the diff into other branches.
```

Comment:
This is mandatory as well, right? It doesn't seem like there's an easy default (unless we search for the most recent PR). So it seems like this ought to be an argument (ideally there's a single argument, so having the fork owner have a default and be an option would allow this to be the one argument).